README.md

```diff
@@ -30,3 +30,43 @@ README.md
   /application-b
   /application-c
 ```
+
+## Run Cucumber tests
+
```

Comment:
Is it possible to add a brief description of what the Cucumber tests are for?

Reply:
Of course, I will add a brief description